tools/pkf2stl2.cpp

```diff
@@ -27,6 +27,24 @@ static void write_facet(ostream &file,
        << el << ef;
 }
 
+static void write_STL(ostream &file, const Tube<Vector3> &tube, const int S) {
```

Comment:
I have seen this one before. Move it into Tube and call it here instead?

Reply:
Yes, this was removed in a later commit so will be fixed at the same time as the other one.